### PR TITLE
Support for rotated ellipses

### PR DESCRIPTION
**Pull request by vascotenner**
```
hv.NdOverlay({angle: hv.Ellipse(0,0,5,aspect=2, angle=angle) for angle in np.linspace(0,np.pi/2, 6)}, kdims=['Angle'])
```

download

### CODE REVIEW

File: holoviews/element/path.py [hunk 1]
```diff
@@ -193,6 +193,8 @@ class Ellipse(BaseShape):
 
     aspect= param.Number(default=1.0, doc="The aspect ratio of the ellipse.")
 
+    angle = param.Number(default=0, doc="The angle in radian between the long axis and the horizontal.")
+
```
Comment (jbednar) [inline holoviews/element/path.py]: To be compatible with our other packages, such a parameter should probably be called `orientation`, and the description could be "Orientation in the Cartesian coordinate system, with zero at 3 o'clock and increasing counterclockwise".

**Comment by philippjfr** [comment]
Apart from Jim's comment this looks like a good addition and I'll be happy to merge once you've renamed the parameter.

**Comment by jlstevens** [comment]
Looks good! 

I agree the docs need to be updated to show an example of an oriented ellipse but that doesn't necessarily have to be part of this PR. I think @philippjfr or myself can easily add such an example the next time we find ourselves updating the website.

I'm also happy to see this merged once you've completed the suggested renaming.

**Comment by philippjfr** [comment]
Looks good. Merging.

**Pull request merged**